### PR TITLE
IDE-715 Add Programmer's Guide Example Code to IDE examples

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>
@GordonSmith Please review

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

File: docs/EN_US/ECLProgrammersGuide/PRG_Mods/PrG_Creating_example_data.xml [hunk 1]
```diff
@@ -8,35 +8,54 @@
     <title>Getting Code Files</title>
 
     <para>All the example code for the <emphasis>Programmer's Guide</emphasis>
-    is available for download from the HPCC Systems website from the same page
-    that the PDF is available (<ulink
-    url="https://hpccsystems.com/training/documentation/learning-ecl">https://hpccsystems.com/training/documentation/learning-ecl</ulink>).
-    To make this code available for use in the ECL IDE, you simply:</para>
+    is installed with the IDE. Please note that it is installed to a read-only
+    folder. To use it, you should copy it to your MyFIles folder. </para>
```
Comment (GordonSmith) [inline docs/EN_US/ECLProgrammersGuide/PRG_Mods/PrG_Creating_example_data.xml]: Typo = My**FI**les

File: docs/EN_US/ECLProgrammersGuide/PRG_Mods/PrG_Creating_example_data.xml [hunk 2]
```diff
@@ -8,35 +8,54 @@
...
+
+    <para>To make this code available for read/write use in the ECL IDE, you
+    simply:</para>
 
     <orderedlist>
       <listitem>
-        <para>Download the ECL_Code_Files.ZIP file.</para>
+        <para>In the ECL IDE, expand the <emphasis
```
Comment (GordonSmith) [inline docs/EN_US/ECLProgrammersGuide/PRG_Mods/PrG_Creating_example_data.xml]: "In the ECL IDE" is duplicated from introduction paragraph

File: docs/EN_US/ECLProgrammersGuide/PRG_Mods/PrG_Creating_example_data.xml [hunk 3]
```diff
@@ -8,35 +8,54 @@
...
-        <para>Name your new folder "ProgrammersGuide" (please note -- spaces
-        are NOT allowed in your ECL repository folder names).</para>
+        <para>From Windows Explorer, copy the <emphasis
+        role="bold">ProgrammersGuide</emphasis> folder. </para>
       </listitem>
 
       <listitem>
-        <para>In the ECL IDE, highlight your "ProgrammersGuide" folder,
-        right-click and select "Locate File in Explorer" from the popup
+        <para>In the ECL IDE, expand the <emphasis role="bold">My
```
Comment (GordonSmith) [inline docs/EN_US/ECLProgrammersGuide/PRG_Mods/PrG_Creating_example_data.xml]: "In the ECL IDE" is duplicated from introduction paragraph

**Review by GordonSmith — CHANGES_REQUESTED** [review]

**Comment by richardkchapman** [comment]
@JamesDeFabia

**Comment by JamesDeFabia** [comment]
@GordonSmith edits made,  back to you

**Review by GordonSmith — APPROVED** [review]
Looks good.

**Comment by richardkchapman** [comment]
@JamesDeFabia Please squash

**Comment by JamesDeFabia** [comment]
@richardkchapman I have squashed  and pushed

**Comment by richardkchapman** [comment]
@JamesDeFabia Can you check the smoketest issue please

**Comment by JamesDeFabia** [comment]
@richardkchapman I think I know the issue. Working on it

**Pull request merged**